src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -22,14 +23,6 @@ import { ChplTextField } from 'components/util';
 import theme from 'themes/theme';
 
 const useStyles = makeStyles({
-  datetimeLayout: {
-    display: 'flex',
-    flexDirection: 'column',
-    [theme.breakpoints.up('md')]: {
```

Comment:
Was this the only place the `theme` import was used? If so, we should be able to remove the import of the `them`